registry/consul.go

```diff
@@ -0,0 +1,55 @@
+package registry
```

Comment:
So if we make this a consul package, inside `lile` then I think that's a boon and we add some others, like `zookeeper` etc

Reply:
I moved it around